microsoft-edge/web-platform/ad-selection-api.md

```diff
@@ -310,16 +311,39 @@ To provide feedback about the Ad Selection API origin trial, create a new issue
 
 * To submit a new issue, go to the [new issue form](https://github.com/WICG/privacy-preserving-ads/issues/new?labels=Ad+Selection%20API,OriginTrialFeedback&title=%5BAd%20Selection%20API%5D+Feedback).
 
+<!-- ====================================================================== -->
+## Terms of Use
+
+* Your use of the Ad Selection API and related features ("Ad API") is subject to the terms contained herein and the [Microsoft APIs Terms of Use](/legal/microsoft-apis/terms-of-use), except that Section 3(b)(12) of the Terms of Use ("Use or transfer any data accessed ... for advertising or marketing purposes") does not apply to your use of the Ad API.
```

Comment:
added:
("Use or transfer any data accessed ... for advertising or marketing purposes")
to confirm which section is 3(b)(12) in case numbering changes.